crates/frontend/src/pages/default_config.rs

```diff
@@ -482,6 +496,90 @@ pub fn get_bread_crums(key_prefix: Option<String>) -> Vec<BreadCrums> {
     bread_crums
 }
 
+#[component]
+fn default_config_filter_widget(
+    filters_rws: RwSignal<DefaultConfigFilters>,
+    key_prefix: RwSignal<Option<String>>,
+) -> impl IntoView {
+    let filters = filters_rws.get_untracked();
+    let filters_buffer_rws = create_rw_signal(filters.clone());
+    view! {
+        <DrawerBtn
+            drawer_id="default_config_filter_drawer".into()
+            style="cursor-pointer btn btn-purple-outline m-1".to_string()
+        >
+            Filters
+            <i class="ri-filter-3-line"></i>
+        </DrawerBtn>
+        <Drawer
+            id="default_config_filter_drawer".to_string()
+            header="default_config Filters"
+            drawer_width="w-[50vw]"
+            handle_close=move || {
+                close_drawer("default_config_filter_drawer");
+            }
+        >
+            <div class="card-body">
+                <div class="flex flex-col gap-1 justify-between">
+                    <div class="form-control">
+                        <label class="label">
+                            <span class="label-text">Configuration Name</span>
+                        </label>
+                        <input
+                            type="text"
+                            id="default-config-name-filter"
+                            placeholder="eg: city"
+                            class="input input-bordered rounded-md resize-y w-full max-w-md"
+                            value=move || {
+                                let prefix = key_prefix.get();
+                                if prefix.is_some() {
+                                    prefix
+                                } else {
+                                    filters_buffer_rws.get().name
+                                }
```

Comment:
shouldn't it be this ?
because if the prefix is some value, but I update the filters value, I would still be showing prefix value itself
```suggestion
                                if prefix.is_some() && filters_buffer_rws.get().name.is_empty() {
                                    prefix
                                } else {
                                    filters_buffer_rws.get().name
                                }
```

Reply:
Good catch